sdk/examples/xo_python/sawtooth_xo/xo_cli.py

```diff
@@ -298,9 +332,17 @@ def do_take(args, config):
     url = config.get('DEFAULT', 'url')
     key_file = config.get('DEFAULT', 'key_file')
 
-    client = XoClient(base_url=url,
-                      keyfile=key_file)
-    response = client.take(name, space)
+    auth_user = args.auth_user
+    auth_password = args.auth_password
+    if auth_user is not None and auth_password is None:
+        auth_password = getpass.getpass(prompt="auth_password: ")
```

Comment:
You use these four lines four times in this file. Would probably be worth abstracting them into a function.